lib/Entity/DisplayGroup.php

```diff
@@ -768,6 +780,7 @@ private function edit()
             'isDynamic' => $this->isDynamic,
             'dynamicCriteria' => $this->dynamicCriteria,
             'dynamicCriteriaTags' => $this->dynamicCriteriaTags,
+            'bandwidthLimit' => $this->bandwidthLimit,
```

Comment:
depending on the answer to the migration question, we might need to protect this from null's